### DISCUSSION TITLE
Different border radius on div corners not works as expected

### DISCUSSION THREAD

**Discussion by erdemgonul** [tailwindlabs/tailwindcss#7402] [Help] [answered]
Hi tailwind guys,

As you see in the Tailwind Play link, different border radius on corners are not working as expected. 

**Case:**

`<div class="bg-red-500 w-10 h-10 rounded-l-full rounded-r-lg"></div>
`

_Div Left Side : rounded-l-full
Div Right Top Side : rounded-tr-lg_

In here, right top side of div is not rounded but with css I can do that correctly.

[Scenario : Tailwind Play Link](https://play.tailwindcss.com/WWgJMarDSH
)

Does anybody have an idea about it?

**Comment by NovusOpus** [comment, marked as answer]
Since you're trying to recreate very specific values the trick is to use Tailwind classes with similar or exact dimensions.

For example for the left border, instead of `rounded-l-full` you can use `rounded-l-3xl` with a size of: `1.5rem` or `24px`.

And for the top-right border: `rounded-tr-2xl` with a size of: `1rem` or `16px`:

```html
<div class="my-4 bg-red-500 w-10 h-10 rounded-l-3xl rounded-tr-2xl"></div>
```

Or you can use [Arbitrary values](https://tailwindcss.com/docs/border-radius#arbitrary-values) for recreating the exact same values:

```html
<div class="my-4 bg-red-500 w-10 h-10 rounded-l-[25px] rounded-tr-[15px]"></div>
```

Example: [Tailwind Play](https://play.tailwindcss.com/uGXJnYzbaO)

**Reply by erdemgonul** [reply]
Wow thank you !

But still couldn't understand why **rounded-l-full** does not work? Isn't it a bug?

**Reply by NovusOpus** [reply]
@erdemgonul The `rounded-l-full` or `rounded-full` value is: `9999px` the "why?" it's explained [here](https://github.com/tailwindlabs/tailwindcss/discussions/3373).

So  `rounded-tr-lg` is being overridden by `rounded-l-full(9999px)` Since there's not enough space (the div is just 40px). 

It's like using this in normal CSS:

```css
border-radius: 9999px 15px 0px 9999px;
```
